tests/handlers/test_freshmaker.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+import json
+from os import path
+
+from estuary.models.freshmaker import FreshmakerEvent
+from estuary.models.errata import Advisory
+
+from estuary.models.koji import ContainerKojiBuild
+from tests import message_dir
+import mock
+from estuary_updater.handlers.freshmaker import FreshmakerHandler
+from estuary_updater import config
+
+
+@mock.patch('koji.ClientSession')
+def test_event_to_building(mock_koji_cs):
+    """Test the Freshmaker handler when it receives an event to building message."""
+    mock_koji_session = mock.Mock()
+    mock_koji_session.getTaskResult.side_effect = [
+        {
+            'koji_builds': ['710916']
+        },
+        {
+            'koji_builds': ['123456']
+        },
+        {
+            'koji_builds': ['234567']
+        }
+    ]
+    mock_koji_cs.return_value = mock_koji_session
+    # Load the message to pass to the handler
+    with open(path.join(message_dir, 'freshmaker', 'event_to_building.json'), 'r') as f:
+        msg = json.load(f)
+    # Make sure the handler can handle the message
+    assert FreshmakerHandler.can_handle(msg) is True
+    # Instantiate the handler
+    handler = FreshmakerHandler(config)
+    # Run the handler
+    handler.handle(msg)
+
+    # Verify everything looks good in Neo4j
+    event = FreshmakerEvent.nodes.get_or_none(id_='2092')
+    assert event is not None
+
+    # Check to see if properties are correct
+    assert event.id_ == '2092'
+    assert event.event_type_id == 8
+    assert event.message_id == ('ID:messaging-devops-broker01.web.prod.ext.phx2.redhat.com-42045-15'
+                                '27890187852-9:704208:0:0:1.RHBA-8018:0593-01')
+    assert event.state == 1
+    assert event.state_name == 'BUILDING'
+    assert event.url == 'http://freshmaker.domain.com/api/1/events/2092'
+
+    # Verify that an advisory also gets created -- search_key is the id of the advisory
+    advisory = Advisory.nodes.get_or_none(id_='34625')
+    assert advisory is not None
+    # Verify that the event.triggered_by_advisory has the relationship to the advisory
+    assert event.triggered_by_advisory.is_connected(advisory)
+
+    # Mock koji client session and getTaskResult
+    build = ContainerKojiBuild.nodes.get_or_none(id_=710916)
```

Comment:
Since you added comments above like `# Verify that an advisory also gets created`, it'd be nice if you added them here too to be consistent. Alternatively you could remove them above since the code is somewhat self-explanatory.